index.html

```diff
@@ -59,8 +59,37 @@
     </a>
 
     <!-- Hero -->
-    <section id="hero" class="section-pad-left">
-
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5 - 6, 2022 | RIT Online Hackathon</h3>
+            <p>Sign up to be notified when applications open: </p>
+            <!-- MailChimp email form, copied from bh8 preregister -->
+            <div class="field-row-stacked">
```

Comment:
This looks bad on mobile, but I don't know if it's intended or not - I can't find a mobile design for this in the Figma.
![image](https://user-images.githubusercontent.com/16471311/144190652-34d7b468-073a-4a48-adcb-5c798fd0b74e.png)

Reply:
Irrelevant because we're removing that element